socorro/external/es/super_search_fields.py

```diff
@@ -472,6 +484,105 @@ def flag_field(
     }
 
 
+def keyword_field(
```

Comment:
I like the feel of these field generating functions and it definitely reduces some risk of copy-and-paste errors. Next step is probably to apply them across all the super search fields and see which field behavior categories we're missing.

Also, it might be useful to add an "override" kind of thing. I'm on the fence about that.